raiden/utils/nursery.py

```diff
@@ -84,7 +84,7 @@ def subprocess_stopped(result: AsyncResult) -> None:
 
                         # if the subprocess error'ed propagate the error.
                         if result.get() != STATUS_CODE_FOR_SUCCESS:
-                            log.error("Process died! Bailing out.")
+                            log.error("Process died! Bailing out.", args=process.args)
```

Comment:
Shouldn't this be `process_args`?

Reply:
Good point, why did this autocomplete?

Reply:
Weird, do you know how `process` gets in scope in line 83?

Reply:
Yes, it's assigned in L94, which is in the same scope and executed first.